app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PaymentsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+  load_resource :conference, find_by: :short_title
+  authorize_resource :conference_registrations, class: Registration
+
+  def index
+    @payments = Payment.where(user_id: current_user.id)
+  end
+
+  def new
+    @total_amount_to_pay = Ticket.total_price(@conference, current_user, 'f')
```

Comment:
Yes, I used to squash the commits with the earlier commit and force-pushed the changes... so I will not squash the commits while repairing changes from now onwards